scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/lfcsp.yaml

```diff
@@ -128,6 +128,73 @@ LFCSP-16-1EP_3x3mm_P0.5mm_EP1.7x1.7mm:
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
 
+
+LFCSP-20-1EP_4x4mm_P0.5mm_EP2.6x2.6mm:
+  device_type: 'LFCSP'
+  library: Package_CSP
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://www.analog.com/media/en/package-pcb-resources/package/pkg_pdf/lfcspcp/cp-20/CP_20_8.pdf'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  # custom_name_format:
+  body_size_x:
+    minimum: 3.9
+    nominal: 4
+    minimum: 4.1
```

Comment:
Should be `maximum`, and also the one below

Reply:
The error was already present in the entries above (which I copied as a start). Fixed it in both